assets/app/scripts/controllers/deployments.js

```diff
@@ -73,6 +94,20 @@ angular.module('openshiftConsole')
       });
     }
 
+    function associateRunningDeploymentToDeploymentConfig(deploymentsByDeploymentConfig) {
+      var deploymentConfigDeploymentsInProgress = {};
+      angular.forEach(deploymentsByDeploymentConfig, function(deploymentConfigDeployments, deploymentConfigName) {
+        deploymentConfigDeploymentsInProgress[deploymentConfigName] = {};
+        angular.forEach(deploymentConfigDeployments, function(deployment, deploymentName) {
+          var deploymentStatus = $scope.deploymentStatus(deployment);
+          if (deploymentStatus === "New" || deploymentStatus === "Pending" || deploymentStatus === "Running") {
+            deploymentConfigDeploymentsInProgress[deploymentConfigName][deploymentName] = deployment;
+          }
+        });
+      });
+      return deploymentConfigDeploymentsInProgress;
+    }
+
```

Comment:
Yeah, it makes sense to me. I would handle it separately (not as part of this PR) but definitely makes sense.